tests/dashbio_demos/app_needle_plot.py

```diff
@@ -103,108 +103,258 @@ def description():
-                                                    {
-                                                        'label': 'Demo dataset',
-                                                        'value': DEMO_KEY
-                                                    },
-                                                    {
-                                                        'label': 'Upload dataset',
-                                                        'value': FILE_KEY
-                                                    },
-                                                    {
-                                                        'label': 'UniProt dataset',
-                                                        'value': DATABASE_KEY
-                                                    },
-                                                ],
-                                                value=DEMO_KEY
-                                            )
+                                        dcc.Input(
+                                            id='needle-sequence-input',
+                                            value='',
+                                            type='text',
+                                            placeholder='TP53, DDX3X, SMARCA4, ...',
+                                        ),
+                                        html.Button(
+                                            id='needle-search-sequence-button',
+                                            children='submit',
+                                            n_clicks=0,
+                                            n_clicks_timestamp=0,
+                                        )
+                                    ]
+                                ),
+                                html.Div(
+                                    id='needle-uniprot-div',
+                                )
+                            ]
+                        ),
+
+                        html.Div(
+                            id='needle-%s-div' % FILE_KEY,
+                            className='needle-load-option-div',
+                            children=[
+                                html.Div(
+                                    id='needle-mutdata-file-div',
+                                    title='Mutation data files are json files containing '
+                                          'the fields :\n'
+                                          '- "x" (protein coordinate of the mutation) \n'
+                                          '- "y" (number of recorded mutations) \n'
+                                          '- "mutationGroups" (type of mutations) \n'
+                                          '- "domains" (protein domains.\n '
+                                          '"x", "y", "mutationGroups" are arrays, they '
+                                          'must have the same length or be empty, '
```

Comment:
```suggestion
                                          'must have the same length, '
```
If they are empty, their length is zero, so it doesn't add information, does it?

Reply:
I think @Bachibouzouk meant that one of them can be empty, and the other of arbitrary length?